app/actions/actionTypes.js

```diff
@@ -1,2 +1,5 @@
 export const LOAD_USER_SUCCESS = 'LOAD_USER_SUCCESS';
 export const GET_USER_SUCCESS = 'GET_USER_SUCCESS';
+export const CREATE_USER_SUCCESS = 'CREATE_USER_SUCCESS';
+export const UPDATE_USER_SUCCESS = 'UPDATE_USER_SUCCESS';
+export const DELETE_USER_SUCCESS = 'DELETE_USER_SUCCESS';
```

Comment:
On this one I'm with @juampick. These constants could be used across several files. But... the ones placed at `apiEndpoints` I'm not so sure. I believe it will be better to delete them because are only being used in one place and it will improve readability. So keep that in mind next time we review a service file (where they are used). Thanks anyway for your feedback @AlphaGit.